serializations.md

```diff
@@ -732,21 +753,58 @@ tree.
 
 ```
 
+###### Channel off-chain pinned block for on-chain environment
+
+Each update can be optionally based on an on-chain block that is used for an
+environment for on-chain lookups: accounts, oracles queries, responses and
+etc. This is to improve the process of both reaching an off-chain conesus
+regarding changes and having more predictable on-chain forced progress
+executions.
+
+```
+[ <hash>      :: binary()
+, <type>      :: int()
+]
+```
+Hash is the block hash to be used.
+Type is the expected block type:
+* 1 stands for a keyblock
+* 2 stands for a microblock
+It is worth mentioning that the pinned block could as well be missing in the
+current longest fork and this will make the transaction that is using it
+invalid.
+
+
 #### Channel off-chain transaction
 
 The channel off-chain transaction is not included directly in the transaction tree but indirectly as payload of:
 * The channel close solo transaction.
 * The channel slash transaction.
 * The channel snapshot solo transaction.
 * The channel force progress transaction.
+version 1: 
```

Comment:
If I read this section, it looks both versions 1 and 2 are legal in Roma and Minerva and Fortuna.